Ability to run in docker (#171)
treetrum/amazon-kindle-bulk-downloader: README.md

Note that amazon credentials will need to be provided. Currently this script expects them to be in the following ENV variables:

| Variable             | Description                        | Required                    |
| -------------------- | ---------------------------------- | --------------------------- |
| `AMAZON_USER`        | Account username                   | Yes                         |
| `PASSWORD`           | Account password                   | Yes                         |
| `OTP`                | 6 digit one-time password          | If 2 factor auth is enabled |
| `PUPPETEER_HEADLESS` | run puppeteer in headless mode     | no (default false)          |

> | `PUPPETEER_ARGS`     | additional arguments for puppeteer | no                          |

Reviewer: I suggest we split these 2 ENV variables out into another section of the README to avoid confusion (Likely part of the docker sections). This step in the README is primarily focused on getting authenticated.

bun run start --baseUrl "https://www.amazon.com.au"
```

### Run in docker

build docker image

```bash
docker build . -t amazon-kindle-bulk-downloader
```

> run container

Reviewer: Can we be a little more descriptive here? I.e.

```suggestion
Run the built docker image ensuring to pass in all the required ENV vars and any CLI flags you wish to override. See below for an example:
```

```bash

> docker run --rm -ti -v ./downloads:/app/downloads -e AMAZON_USER=userName -e PASSWORD=pass -e OTP=otpCode amazon-kindle-bulk-downloader --baseUrl "https://www.amazon.com"

Reviewer: Can we split up this command over multiple lines so it reads a little easier and without having to horizontally scroll.

I.e.

```suggestion
docker run --rm -ti \
    -v ./downloads:/app/downloads \
    -e AMAZON_USER=userName \
    -e PASSWORD=pass \
    -e OTP=otpCode \
    amazon-kindle-bulk-downloader \
    --baseUrl "https://www.amazon.com"
```

```

> In docker `--manualAuth` does not work you must provide credentials via env

Reviewer: Can we nest these points under an `#### Important Notes` heading?

> If you are on arm64 (ex Mac with apple silicone) you mast add `--platform linux/x86_64` to thous commands

Reviewer: ```suggestion
If you are on arm64 (i.e. a Mac with an Apple Silicon chip) you mast add `--platform linux/x86_64` when running your `docker run` command.
```
